homeassistant/components/heos/strings.json

```diff
@@ -20,13 +20,30 @@
         "data_description": {
           "host": "[%key:component::heos::config::step::user::data_description::host%]"
         }
+      },
+      "reauth_confirm": {
+        "title": "Reauthenticate HEOS",
+        "description": "Please update your HEOS Account credentials. Alternatively, the existing credentials can be cleared to complete this action.",
```

Comment:
> to complete this action.
Can we be more specific to what that means? What does the user loose if they remove their credentials?